include/itkOMEZarrNGFFImageIO.h

```diff
@@ -231,6 +239,11 @@ class IOOMEZarrNGFF_EXPORT OMEZarrNGFFImageIO : public ImageIOBase
   int                m_TimeIndex = INVALID_INDEX;
   int                m_ChannelIndex = INVALID_INDEX;
   AxesCollectionType m_StoreAxes;
+
+  const static unsigned m_EmptyZipSize = 22;
```

Comment:
Could you please add a comment clarifying how this value is derived?

Not required, but could rename to reflect that this is a static constant: `static constexpr unsigned EMPTY_ZIP_SIZE = 22;`